### DISCUSSION TITLE
Multiple Rows in a Cell

### DISCUSSION THREAD

**Discussion by rezlab** [icflorescu/mantine-datatable#275] [Q&A]
Is there any way to display multiple values in a given cell on different lines? I have tried a few approaches such as inserting "\<br/>" and '/n' between them.

I am looking to display something in a given row for a given column
Task A
Task B
Task C.

What I seem to get is:
Task A Task B Task C.

**Comment by icflorescu** [comment]
You can **absolutely** display on different lines in a given cell.
The column `render` method accepts the current record and must return a `ReactNode`.
In the simplest usage scenario you could have a column definition like so:
```ts
{
  accessor: 'name',
  render: ({ firstName, lastName }) => (
    <>
      {firstName}
      <br />
      {lastName}
    </>
  ),
},
// ...
```
